test/unit/uk/gov/hmrc/bindingtariffrulingfrontend/model/PagedTest.scala

```diff
@@ -26,28 +28,130 @@ class PagedTest extends BaseSpec {
     }
 
     "calculate size" in {
-      Paged.empty.size    shouldBe 0
-      Paged(Seq("")).size shouldBe 1
+      Paged.empty.size                     shouldBe 0
+      Paged.empty(SimplePagination()).size shouldBe 0
+      Paged(Seq("")).size                  shouldBe 1
     }
 
     "calculate isEmpty" in {
-      Paged.empty.isEmpty    shouldBe true
-      Paged(Seq("")).isEmpty shouldBe false
+      Paged.empty.isEmpty                     shouldBe true
+      Paged.empty(SimplePagination()).isEmpty shouldBe true
+      Paged(Seq("")).isEmpty                  shouldBe false
     }
 
+    // scalastyle:off magic.number
     "calculate pageCount" in {
       Paged.empty.pageCount shouldBe 0
-      Paged(results = Seq(), pageIndex = 1, pageSize = 1, resultCount  = 1).pageCount   shouldBe 1
-      Paged(results = Seq(), pageIndex = 1, pageSize = 1, resultCount  = 2).pageCount   shouldBe 2
-      Paged(results = Seq(), pageIndex = 1, pageSize = 2, resultCount  = 3).pageCount   shouldBe 2
-      Paged(results = Seq(), pageIndex = 1, pageSize = 10, resultCount = 100).pageCount shouldBe 10
-      Paged(results = Seq(), pageIndex = 1, pageSize = 1, resultCount  = 100).pageCount shouldBe 100
+      Paged(results = Seq(), pageIndex   = 1, pageSize = 1, resultCount = 1).pageCount shouldBe 1
+      Paged(results = Seq(), pageIndex   = 1, pageSize = 1, resultCount = 2).pageCount shouldBe 2
+      Paged(results = Seq(), pageIndex   = 1, pageSize = 2, resultCount = 3).pageCount shouldBe 2
+      Paged(results = Seq(), pageIndex   = 1, pageSize = 10, resultCount = 100).pageCount shouldBe 10
+      Paged(results = Seq(), pageIndex   = 1, pageSize = 1, resultCount = 100).pageCount shouldBe 100
+      Paged(results = Seq(), resultCount = 1).pageCount shouldBe 1
+      Paged(results = Seq(), resultCount = 100).pageCount shouldBe 1
+      Paged(results = Seq(), resultCount = 1000).pageCount shouldBe 10
     }
+    // scalastyle:on magic.number
 
     "calculate nonEmpty" in {
-      Paged.empty.nonEmpty    shouldBe false
-      Paged(Seq("")).nonEmpty shouldBe true
+      Paged.empty.nonEmpty                     shouldBe false
+      Paged.empty(SimplePagination()).nonEmpty shouldBe false
+      Paged(Seq("")).nonEmpty                  shouldBe true
     }
+
+    "valid results when results are provided" in {
+      val validPaged = Json
```

Comment:
all this copy paste around of json paged object its a bit ugly, you can extract this logic into a method and have all 4 as params inside with some defaults.
For example:
```val results = Seq("")
val pageIndex = 1
val pageSize = 1
val resultCount = 1
val pagedResult = buildJson(results, pageIndex,pageSize,resultCount).as[Paged[String]]
val expectedPagedResult = buildPaged(results, pageIndex,pageSize,resultCount)

pagedResult shouldBe expectedPagedResult
```
Feel free to change structure or method names to make it more readable or shorter. This can be re-used/applied to most test cases I see here